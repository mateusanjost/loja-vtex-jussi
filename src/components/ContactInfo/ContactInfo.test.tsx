import TestRenderer from "react-test-renderer";
import ContactInfoComponent from "./ContactInfo";

describe("Contact Info", () => {
  it("should match snapshot", () => {
    const testRenderer = TestRenderer.create(<ContactInfoComponent />);
    const jsonComponent = testRenderer.toJSON();

    expect(jsonComponent).toMatchSnapshot();
  });
});
