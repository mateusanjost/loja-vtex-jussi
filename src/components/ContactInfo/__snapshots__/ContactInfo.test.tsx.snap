// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact Info should match snapshot 1`] = `
<div
  className="sc-hKFxyN sc-eCApnc okUlG fQyHhC"
>
  <div
    className="sc-hKFxyN sc-iCoGMd okUlG dNAIIn"
  >
    <div
      className="sc-gtsrHT sc-jSFjdj lejSiq ghvjZO"
    >
      Essa loja foi construída usando uma das nossas soluções da plataforma VTEX. tenha a sua.
    </div>
    <div
      className="sc-gtsrHT sc-jSFjdj lejSiq ghvjZO"
    >
       
      <div
        className="sc-bdnxRM sc-gKAaRy cHGDwy"
      >
        Entre em contato
      </div>
      <a
        href="mailto:comercial@jussi.com.br"
        rel="noopener noreferrer"
        target="_blank"
      >
        comercial@jussi.com.br
      </a>
    </div>
  </div>
</div>
`;
